file-formats/uiad/type/zif_aff_uiad_v1.intf.abap

```diff
@@ -0,0 +1,616 @@
+
+      "! <p class="shorttext">Target ID</p>
+      "! Target ID
+      "! $required
+      target_id TYPE c LENGTH 10,
+
+    END OF ty_app_details_web_client.
+
+  TYPES:
+    "! <p class="shorttext">Url Template Parameter</p>
+    "! Url Template parameter
+    BEGIN OF ty_url_template_parameter,
+
+      "! <p class="shorttext">Name</p>
+      "! Parameter name
+      name  TYPE string,
+
+      "! <p class="shorttext">Value</p>
+      "! Parameter value
+      value TYPE string,
+
+    END OF ty_url_template_parameter,
+
+    ty_url_template_parameters TYPE STANDARD TABLE OF ty_url_template_parameter WITH DEFAULT KEY.
+
+  TYPES:
+    "! <p class="shorttext">Url Template App Details</p>
+    "! Url Template specific app details
+    BEGIN OF ty_app_details_url_template,
+
+      "! <p class="shorttext">Template ID</p>
+      "! Url Template ID
+      "! $required
+      template_id TYPE c LENGTH 32,
+
+      "! <p class="shorttext">Parameters</p>
+      "! Url Template parameters
+      parameters  TYPE ty_url_template_parameters,
+
+    END OF ty_app_details_url_template.
+
+  "! <p class="shorttext">App Type</p>
+  "! LADI app type
+  "! $values {@link zif_aff_uiad_v1.data:co_app_type}
+  TYPES ty_app_type TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">App Type</p>
+    "! LADI app type
+    BEGIN OF co_app_type,
+
+      "! <p class="shorttext">Transaction</p>
+      "! Transaction
+      transaction  TYPE ty_app_type VALUE 'T',
+
+      "! <p class="shorttext">Web Dynpro Application</p>
+      "! Web Dynpro application
+      web_dynpro   TYPE ty_app_type VALUE 'W',
+
+      "! <p class="shorttext">Web Client UI Application</p>
+      "! Web Client UI application
+      web_client   TYPE ty_app_type VALUE 'C',
+
+      "! <p class="shorttext">UI5</p>
+      "! SAPUI5 Fiori app
+      ui5          TYPE ty_app_type VALUE 'U',
+
+      "! <p class="shorttext">SAPUI5 Fiori App on SAP BTP</p>
+      "! SAPUI5 Fiori app on SAP BTP (Deprecated)
+      legacy_ui5   TYPE ty_app_type VALUE 'S',
+
+      "! <p class="shorttext">URL App</p>
+      "! URL app
+      url          TYPE ty_app_type VALUE 'R',
+
+      "! <p class="shorttext">Tile Only</p>
+      "! Tile only
+      tile         TYPE ty_app_type VALUE 'O',
+
+      "! <p class="shorttext">Url Template</p>
+      "! Url Template
+      url_template TYPE ty_app_type VALUE 'G',
+
+    END OF co_app_type.
+
+  TYPES:
+    "! <p class="shorttext">App Details</p>
+    "! Technology specific app details
+    BEGIN OF ty_app_details,
+
+      "! <p class="shorttext">App Type</p>
+      "! LADI app type
+      "! $required
+      app_type     TYPE ty_app_type,
+
+      "! <p class="shorttext">System Alias</p>
+      "! System alias
+      system_alias TYPE c LENGTH 32,
+
+      "! <p class="shorttext">UI5 App Details</p>
+      "! UI5 specific app details
+      ui5          TYPE ty_app_details_ui5,
+
+      "! <p class="shorttext">WDA App Details</p>
```

Comment:
Do you want to keep this title or also change it to Web Dynpro App Details?

Reply:
Sorry missed it